### DISCUSSION TITLE
NBM CONUS Snowfall

### DISCUSSION THREAD

**Discussion by tmabell** [open-meteo/open-meteo#1136] [Q&A] [answered]
I'm not seeing any snowfall forecasted amounts from the NBM being returned by the API .  I've checked multiple locations where there should be (and is) accumulating snow and the API always returns 0".

**Comment by patrick-zippenfenig** [comment]
Hi. Forecasts should contain snowfall, however archived data is for yet unknown reasons not correct. See: https://github.com/open-meteo/open-meteo/issues/1123

**Comment by tmabell** [comment]
Thanks for your response but while it should contain snowfall, I'm reasonably certain that it doesn't.  Look at International Falls, MN for Tuesday 3 DEC.  Remember that the NBM is a blend of multiple models and as such is going to show something when all of the other models do.  
https://open-meteo.com/en/docs#latitude=48.6011&longitude=-93.411&hourly=&daily=snowfall_sum&precipitation_unit=inch&timezone=America%2FNew_York&models=ecmwf_ifs025,ecmwf_aifs025,bom_access_global,gfs_seamless,gfs_global,ncep_nbm_conus,icon_global,gem_seamless,gem_global,ukmo_global_deterministic_10km

and compare to the Medium-range NBM MOS product NBE...
https://mesonet.agron.iastate.edu/wx/afos/#NBEINL:1

**Reply by patrick-zippenfenig** [reply]
I will have a closer look at NBM later today. I also got a report that [archived NBM temperature](https://open-meteo.com/en/docs/gfs-api#latitude=46.25&longitude=-74.58&past_days=61&start_date=2024-11-15&end_date=2024-12-05&time_mode=time_interval&models=gfs_global,gfs_hrrr,ncep_nbm_conus) data has an issue.

**Reply by tmabell** [reply]
Thanks for looking into this Patrick.  I found a little better example if this helps at all:

https://open-meteo.com/en/docs#latitude=47.1573&longitude=-67.9353&hourly=&daily=snowfall_sum&precipitation_unit=inch&timezone=America%2FNew_York&models=ecmwf_ifs04,ecmwf_ifs025,ecmwf_aifs025,bom_access_global,gfs_seamless,gfs_global,ncep_nbm_conus,gfs_graphcast025,icon_global,gem_seamless,gem_global,gem_regional,gem_hrdps_continental,ukmo_global_deterministic_10km

**Reply by patrick-zippenfenig** [reply]
Haven't had time yet to analyse the issue. Maybe tomorrow. Sorry!

**Reply by patrick-zippenfenig** [reply, marked as answer]
A fix is applied here: https://github.com/open-meteo/open-meteo/commit/b4c99cb6fd482c89f0c6ebe1b3e5fc0a94bb0284. The `ASNOW` GRIB field is already in metres. I expected it to be millimetre and snow water equivalent. I am deploying a fix right now to the server processing NBM and tomorrow a fix to the API servers.

**Reply by tmabell** [reply]
Many thanks Patrick!
